README.md

```diff
@@ -216,7 +217,7 @@ def xml_to_pdf():
             'driver': 'xml',
             'xml_xpath': '/CancelResponse/CancelResult/ID',
         },
-        locale='pt_BR'  # LOCALE Ex.:(en_US, de_GE)
+        locale='pt_BR'  # LOCALE Ex.:(en, de)
```

Comment:
Not applicable.
The correct is is in the following en_US format, de_GE, pt_BR ...